packages/application-shell/src/components/user-settings-menu/user-settings-menu.js

```diff
@@ -62,136 +68,207 @@ function stateReducer(state, changes) {
+                  {renderLabel(menu, props.locale)}
+                </Spacings.Inset>
+              </div>
+            </Link>
+          </OptionalFeatureToggle>
+        ))}
+        <div
+          className={classnames(
+            styles.item,
+            styles['item-divider-account-section']
+          )}
+        />
+        <a
+          href={`https://commercetools.com/privacy`}
+          target="_blank"
+          onClick={props.downshiftProps.toggleMenu}
+        >
+          <div className={styles.item}>
+            <Spacings.Inset scale="s">
+              <FormattedMessage {...messages.privacyPolicy} />
+            </Spacings.Inset>
+          </div>
+        </a>
+        <a
+          href={MCSupportFormURL}
+          rel="noopener noreferrer"
+          target="_blank"
+          data-track-component="Navigation-Support-links"
+          data-track-event="click"
+          data-track-label="support_textlink"
+          onClick={props.downshiftProps.toggleMenu}
+        >
+          <div className={styles.item}>
+            <Spacings.Inset scale="s">
+              <FormattedMessage {...messages.support} />
+            </Spacings.Inset>
+          </div>
+        </a>
+        <div
+          className={classnames(
+            styles.item,
+            styles['item-divider-account-section']
           )}
-        </Downshift>
+        />
+        <a
+          // NOTE: we want to redirect to a new page so that the
+          // server can remove things like cookie for access token.
+          href={`/logout?reason=${LOGOUT_REASONS.USER}`}
+          data-test="logout-button"
+        >
+          <div className={styles.item} tabIndex="0">
+            <Spacings.Inset scale="s">
+              <FormattedMessage {...messages.logout} />
+            </Spacings.Inset>
+          </div>
+        </a>
       </div>
-    );
-  }
-}
+    </Card>
+  );
+};
+UserSettingsMenuBody.displayName = 'UserSettingsMenuBody';
+UserSettingsMenuBody.propTypes = {
+  locale: PropTypes.string.isRequired,
+  firstName: PropTypes.string,
+  lastName: PropTypes.string,
+  email: PropTypes.string.isRequired,
+  gravatarHash: PropTypes.string.isRequired,
+  downshiftProps: PropTypes.shape({
+    toggleMenu: PropTypes.func.isRequired,
+    getMenuProps: PropTypes.func.isRequired,
+  }).isRequired,
+  // graphql
+  applicationsMenuQuery: PropTypes.shape({
+    applicationsMenu: PropTypes.shape({
+      appBar: PropTypes.arrayOf(PropTypes.object).isRequired,
+    }),
+  }),
+};
+
+const ConnectedUserSettingsMenuBody = compose(
+  graphql(FetchApplicationsMenu, {
+    name: 'applicationsMenuQuery',
+    skip: ownProps => !!ownProps.DEV_ONLY__getMenuConfig,
+    options: () => ({
+      // We can assume here that the navbar already fetched the data, since this
+      // component gets rendered only when the user opens the menu
+      fetchPolicy: 'cache-only',
+      context: {
+        uri: `${window.location.origin}/api/graphql`,
```

Comment:
Should this maybe be an environment variable? `process.env.MC_PORXY_API_URL` or something?

Reply:
I would keep this as default, we could allow to optionally overwrite it if necessary.

Reply:
Okay.